includes/components/class-user.php

```diff
@@ -55,18 +55,42 @@ public function __construct( $args = [] ) {
 
 			add_action( 'personal_options_update', [ $this, 'update_profile_fields' ], 100 );
 			add_action( 'edit_user_profile_update', [ $this, 'update_profile_fields' ], 100 );
+
+			// Set user status.
+			add_action( 'admin_init', [ $this, 'set_user_status' ] );
```

Comment:
этот хук выглядит не нужным, ведь на бекенде не показываем статус юзера

Reply:
если его убрать, тогда если вендор будет ходить по админке то его онлайн-статус не будет обновляться, а будет обновляться только когда зайдёт на фронт-страницу

Reply:
It's ok to remove this because in 99% of installations access to the back-end for vendors is blocked anyway (this is the default setting in HivePress/Settings/Users).